s-c-s/compromises: contributing some new stories plus a comment for the README (#324)
cncf/tag-security: supply-chain-security/compromises/2003/debian.md

# Rootkit installed in several Debian infrastructure servers

> A sniffed password was used to log into a Debian server, escalate via an unpatched kernel exploit and install a rootkit in at least four different Debian machines.

Reviewer: ```suggestion
A sniffed password was used to log into a Debian server, then privileges were escalated and a rootkit installed in at least four different Debian machines.
```

## Impact

* Debian development slowed down as LDAP/SSH were disabled and secrets rotated
* External checksum lists were used to verify that packages weren't affected

## Type of compromise

> Infrastructure takeover

Reviewer 2: Could you add `(Publishing Infrastructure)` here? so as to match the [definitions here](https://github.com/cncf/sig-security/blob/master/supply-chain-security/compromises/compromise-definitions.md)

Reviewer: ```suggestion
Publishing Infrastructure
```
